plugins/org.jkiss.dbeaver.ext.tidb/plugin.xml

```diff
@@ -0,0 +1,167 @@
+                    </folder>
+                    <folder type="org.jkiss.dbeaver.ext.mysql.model.MySQLCharset" label="%tree.charsets.node.name" icon="#info" description="Database charsets">
+                        <items label="%tree.charset.node.name" path="charsets" property="charsets" icon="#info">
+                            <items label="%tree.collation.node.name" path="collations" property="collations" icon="#info" navigable="false"/></items>
+                    </folder>
+                    <folder type="org.jkiss.dbeaver.model.access.DBAPrivilege" label="%tree.user_privileges.node.name" icon="#info" description="User privileges">
+                        <items label="%tree.privilege.node.name" path="privileges" property="privileges" icon="#info" navigable="false" virtual="true"/>
+                    </folder>
+                </folder>
+            </tree>
+            <drivers managable="true">
+                 <driver
+                    id="tidb"
+                    label="TiDB"
+                    icon="icons/tidb_icon.png"
+                    iconBig="icons/tidb_icon_big.png"
+                    class="com.mysql.cj.jdbc.Driver"
+                    sampleURL="jdbc:mysql://{host}[:{port}]/[{database}]"
+                    defaultPort="4000"
+                    defaultUser="root"
+                    webURL="https://pingcap.com/"
+                    propertiesURL="https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-configuration-properties.html"
+                    description="TiDB Driver (MySQL Connector/J 8.0)"
+                    promoted="1"
+                    categories="sql">
+
+                    <file type="jar" path="maven:/mysql:mysql-connector-java:RELEASE" bundle="!drivers.mysql"/>
+                    <file type="license" path="drivers/mysql/LICENSE.txt" bundle="drivers.mysql"/>
+                    <file type="jar" path="drivers/mysql/mysql8" bundle="drivers.mysql"/>
+
+                    <property name="rewriteBatchedStatements" value="true"/>
+
+                    <parameter name="serverType" value="tidb"/>
+                    <property name="loginTimeout" value="20000"/>
+                    <property name="connectTimeout" value="20000"/>
+
+                    <property name="enabledTLSProtocols" value="TLSv1.2,TLSv1.3"/>
+                    <property name="@dbeaver-default-resultset.maxrows.sql" value="true"/>
+                    <property name="@dbeaver-default-resultset.format.datetime.native" value="true"/>
+                    <property name="@dbeaver-default-dataformat.type.timestamp.pattern" value="yyyy-MM-dd HH:mm:ss"/>
+                </driver>
+
+            </drivers>
+        </datasource>
+    </extension>
+    
+    <extension point="org.jkiss.dbeaver.dataTypeProvider">
+        <provider
+            class="org.jkiss.dbeaver.ext.tidb.data.TiDBValueHandlerProvider"
```

Comment:
Ok, you have this class. But it is empty. What for then?

Reply:
Used `org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCStandardValueHandlerProvider` instead.